src/bits/bit_array.rs

```diff
@@ -303,5 +273,6 @@ macro_rules! bit_array_impl {
     };
 }
 
-bit_array_impl!(bit_array_64, BitArray64, U8_8, 64);
-bit_array_impl!(bit_array_40, BitArray40, U8_5, 40);
+use sha2::digest::typenum::{U5, U8};
```

Comment:
Do you really have to use the sha2 crate for this?  Can we get this from the typenum crate directly?

Reply:
yes that was the plan if this is the right direction